mono/mini/il2cpp-compat.h

```diff
@@ -248,6 +248,7 @@
 #define mono_class_get_element_class il2cpp_class_get_element_class
 #define mono_class_get_type_token il2cpp_class_get_type_token
 #define mono_type_is_byref il2cpp_type_is_byref
+#define mono_type_get_attrs il2cpp_type_get_attrs
```

Comment:
@joshpeterson or @brianradunity does this need something other than this method existing in `il2cpp_type_get_attrs`?

Reply:
No, we should only need `il2cpp_type_get_attrs` on the embedding API. I don't see it on the `master` branch in the IL2CPP repo. Although, we should only need this define here if the debugger code calls `mono_type_get_attrs`.

@enekoosia: Did you see this code called from the debugger-agent.c file?

Reply:
I've just made the pull request in il2cpp to add the il2cpp_type_get_attrs function

Reply:
@joshpeterson No, nobody is calling mono_type_get_attrs yet. I have a branch where I will be using that call in the MonoScriptCache.cpp

Reply:
@enekoosia: Thanks, that sounds good! Let's make sure the IL2CPP pull request lands before this one does, so that the implementation of `il2cpp_type_get_attrs` is there and ready.

Reply:
Ok, no problem ;)